cranelift/codegen/src/ir/extfunc.rs

```diff
@@ -51,8 +51,8 @@ impl Signature {
     }
 
     /// Return an object that can display `self` with correct register names.
-    pub fn display<'a, R: Into<Option<&'a RegInfo>>>(&'a self, regs: R) -> DisplaySignature<'a> {
-        DisplaySignature(self, regs.into())
+    pub fn display(&self) -> DisplaySignature<'_> {
```

Comment:
Can you directly implement Display for Signature?

Reply:
Indeed, thanks!